### ISSUE TITLE
LORA - Resume from saved training state (path to "last-state" state folder)

### ISSUE THREAD

**Issue by kalle07** [bmaltais/kohya_ss#2384]
windows10, RTX4660,

kohya_ss v23.1.4

i can only say it doesnt train from resume state, i see it on tensorboard and on the generated images all 100steps!
all is running  - no error messages - so i cant  tell you anymore ...

**Comment by bmaltais** [comment]
Can you see the path in the toml file? Maybe there is an issue with how I provide this value to the sd-scripts... can you share the toml ?

**Comment by kalle07** [comment]
i searched for "toml" in folder kohya and in the train folder ... nothing  ;)

**Comment by bmaltais** [comment]
It should be in the outputs folder inside the kohya_ss folder.

**Comment by kalle07** [comment]
no empty, but i break the last training...

for me was also a bug, if i restart a training all "ALL" (ok not the Sample prompt) ... all all settings the same.... learning rate, etc
why if i load a restart these parameters not greyed out ?  ;)

**Comment by bmaltais** [comment]
> no empty, but i break the last training...
> 
> for me was also a bug, if i restart a training all "ALL" (ok not the Sample prompt) ... all all settings the same.... learning rate, etc why if i load a restart these parameters not greyed out ? ;)

I absolutly has to be there... as this is where the temporary toml file is created and then called bu the sd-scripts to get the training parameters...

Is there an outputs folder in your kohya_ss folder?

![image](https://github.com/bmaltais/kohya_ss/assets/7474674/04368502-5d79-42d9-b47b-a8ba0b09427b)

**Comment by kalle07** [comment]
sry yes folder is there but empty...
i only checked "Save training state at end of training" and break the training at 1%
also if i check "Save training state" and set above (a bit fare away ;)  )  "Save every N steps" 10 ,  no files in that folder

 v23.1.4

... startlog
21:40:47-803530 INFO     Kohya_ss GUI version: v23.1.4
21:40:49-392214 INFO     Submodule initialized and updated.
21:40:49-392214 INFO     nVidia toolkit detected
21:40:53-802332 INFO     Torch 2.1.2+cu118
21:40:53-871324 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8905
21:40:53-871324 INFO     Torch detected GPU: NVIDIA GeForce RTX 4060 Ti VRAM 16380 Arch (8, 9) Cores 34
21:40:53-871324 INFO     Python version is 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit
                         (AMD64)]
21:40:53-886945 INFO     Verifying modules installation status from requirements_pytorch_windows.txt...
21:40:53-886945 INFO     Verifying modules installation status from requirements_windows.txt...
21:40:53-886945 INFO     Verifying modules installation status from requirements.txt...
21:40:59-521401 INFO     headless: False
Running on local URL:  http://127.0.0.1:7860

**Comment by bmaltais** [comment]
I think this is something kohya might have to look into as this is breaking internally in his sd-scripts and inside the GUI.

**Comment by kalle07** [comment]
ok thx, so its not related on that i dont have the newest version...
